lib/ReactViews/StandardUserInterface/MapColumn.jsx

```diff
@@ -70,8 +70,6 @@ const MapColumn = observer(
                   `}
                 >
                   <MenuBar
-                    terria={this.props.viewState.terria}
-                    viewState={this.props.viewState}
```

Comment:
Has removing this led to the console errors? I.e.
```
Warning: Failed prop type: The prop `viewState` is marked as required in `WithControlledVisibility`, but its value is `undefined`.
```

Reply:
Console error was due to me not updating propTypes. Now fixed.